src/tools/w3dviewer/emittergeneralproppage.h

```diff
@@ -0,0 +1,63 @@
+/**
+ * @file
+ *
+ * @author Jonathan Wilson
+ *
+ * @brief W3D View emitter general prop page
+ *
+ * @copyright Thyme is free software: you can redistribute it and/or
+ *            modify it under the terms of the GNU General Public License
+ *            as published by the Free Software Foundation, either version
+ *            2 of the License, or (at your option) any later version.
+ *            A full copy of the GNU General Public License can be found in
+ *            LICENSE
+ */
+#pragma once
+#include "w3dafx.h"
+#include "shader.h"
+
+class EmitterInstanceList;
+class ColorBarClass;
+class EmitterPropertySheetClass;
+
+class EmitterGeneralPropPageClass : public CPropertyPage
+{
+public:
+    EmitterGeneralPropPageClass();
+    virtual ~EmitterGeneralPropPageClass() override {}

```

Comment:
I recommend to implement this and simply set
```
m_instanceList = nullptr;
m_sheet = nullptr;
```
as a cheap way to document that nothing is leaked as it does not own these.